vike-vue/renderer/onRenderHtml.ts

```diff
@@ -1,19 +1,17 @@
 export default onRenderHtml
 
-import { renderToNodeStream } from '@vue/server-renderer'
-import { escapeInject } from 'vite-plugin-ssr/server'
+import { renderToString } from '@vue/server-renderer'
+import { dangerouslySkipEscape, escapeInject } from 'vite-plugin-ssr/server'
 import { getTitle } from './getTitle.js'
 import type { PageContextServer } from './types'
 import { createVueApp } from './app'
 
 async function onRenderHtml(pageContext: PageContextServer) {
-  let stream: ReturnType<typeof renderToNodeStream> | string
-  if (pageContext.Page === undefined) {
-    // SSR is disabled (SPA mode)
-    stream = ''
-  } else {
+  let pageHtml = ''
+  if (pageContext.Page !== undefined) {
+    // SSR is enabled
     const app = createVueApp(pageContext)
-    stream = renderToNodeStream(app)
+    pageHtml = await renderToString(app)
```

Comment:
I switched to `renderToString` because `escapeInject` isn't happy when getting two streams. Is it OK?

Reply:
I think we can render the head to a string while rendering the body as a stream.

The rationale being that things like `<title>` are important for SEO so it's probably best if it's part of the very first chunk of the HTML stream.